### PR TITLE
migrate pubsub adapter to ssdc, keeping only eq receipting

### PR DESCRIPTION
**Pull request by hbrace**
WHY?
We want to migrate the pubsub adapter to ssdc so that it can accept eq receipts.

WHAT?
Migrated everything from the census pubsub adapter, stripping out all the processors except eq receitping.

HOW?
`make docker` in this repo
`make up` in ssdc rm docker dev on the branch linked from this ticket
`make test` in ssdc rm acceptance tests on the branch linked from this ticket

LINKS?
https://trello.com/c/gdrW9gzU/2418-few-more-repos-to-migrate-pubsub-adapter-data-exporter-5

### CODE REVIEW

**Comment by hbrace** [comment]
[Few more repos to migrate: pubsub adapter, data exporter (5)](https://trello.com/c/gdrW9gzU/2418-few-more-repos-to-migrate-pubsub-adapter-data-exporter-5)

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+script:
+  - make format-check
+  - make build
+  - make test
+  - make docker
+
+branches:
+  only:
+    - master
```
Comment (AdamHawtin) [inline .travis.yml]: Needs to be `main` on new repos

File: config/config.go [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+	RabbitPort                       string `envconfig:"RABBIT_PORT" required:"true"`
+	RabbitUsername                   string `envconfig:"RABBIT_USERNAME" required:"true"`
+	RabbitPassword                   string `envconfig:"RABBIT_PASSWORD"  required:"true"  json:"-"`
+	RabbitVHost                      string `envconfig:"RABBIT_VHOST"  default:"/"`
+	RabbitConnectionString           string `json:"-"`
+	EventsExchange                   string `envconfig:"RABBIT_EXCHANGE"  default:"events"`
+	ReceiptRoutingKey                string `envconfig:"RECEIPT_ROUTING_KEY"  default:"event.response.receipt"`
+	UndeliveredRoutingKey            string `envconfig:"UNDELIVERED_ROUTING_KEY"  default:"event.fulfilment.undelivered"`
+	FulfilmentConfirmationRoutingKey string `envconfig:"FULFILMENT_CONFIRMATION_ROUTING_KEY"  default:"event.fulfilment.confirmation"`
+	FulfilmentRequestRoutingKey      string `envconfig:"FULFILMENT_REQUEST_ROUTING_KEY"  default:"event.fulfilment.request"`
```
Comment (AdamHawtin) [inline config/config.go]: Unused config

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+sudo: required
+
+services:
+  - docker
+
+langauge: go
+
+go:
+  - 1.14.x
```
Comment (AdamHawtin) [inline .travis.yml:9]: This should probably be updated to `1.16.x`, that's what is currently being used in the docker build.

**Review by NickGrantX — APPROVED** [review]

File: .travis.yml (hunk 3 above)
Comment (AdamHawtin) [reply .travis.yml:9]: Ah, if it's causing issues then revert it for now, but I'll make a card to look at it again, it's an inherited problem.

**Review by AdamHawtin — APPROVED** [review]
👍 Looks good, I'll raise a card for the inherited issues

**Pull request merged**